apps/condo/domains/notification/schema/DisconnectUserFromDeviceService.test.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Generated by `createservice notification.DisconnectUserFromDeviceService --type mutations`
+ */
+const { makeLoggedInAdminClient, makeClient, makeLoggedInClient } = require('@core/keystone/test.utils')
+
+const { disconnectUserFromDeviceByTestClient, syncDeviceByTestClient, Device: DeviceAPI } = require('@condo/domains/notification/utils/testSchema')
+
+const { getRandomTokenData } = require('../utils/testSchema/helpers')
+
+describe('DisconnectUserFromDeviceService', () => {
```

Comment:
Please, test a case, when device is not found.

Reply:
fixed